### DISCUSSION TITLE
Is there already a golang client in development?

### DISCUSSION THREAD

**Discussion by fmoor** [geldata/gel#1826] [Q&A] [answered]
I recently started learning golang. I chose to write an edgedb client as a project to motivate my learning. Is there already a golang client in development? It sounds like there might be based on [this comment](https://github.com/edgedb/edgedb/issues/643#issuecomment-490052007). I'm curious if there is room for contribution.

**Comment by 1st1** [comment, marked as answer]
No, unfortunately we haven't yet started working on that, so the room is open :)

**Reply by fmoor** [reply]
Thanks @1st1 for getting back to me! I just published [my work so far](https://github.com/fmoor/edgedb-golang). If the edgedb team has any feedback I'm all ears.

**Reply by 1st1** [reply]
This looks great! Can you ping me at yury@edgedb.com?

**Reply by fmoor** [reply]
pinged :heavy_check_mark:
